app/src/main/java/dev/arkbuilders/rate/presentation/quick/QuickScreen.kt

```diff
@@ -379,11 +382,20 @@ private fun QuickItem(
                     }
                 } else {
                     Text(
-                        text = "${CurrUtils.prepareToDisplay(quick.pair.amount)} ${quick.pair.from} = " +
+                        text = "${CurrUtils.prepareToDisplay(quick.amount)} ${quick.from} = " +
                                 "${CurrUtils.prepareToDisplay(quick.to.first().value)} ${quick.to.first().code}",
                         color = ArkColor.TextTertiary
                     )
                 }
+                Text(
+                    modifier = Modifier.padding(top = if (expanded) 8.dp else 0.dp),
+                    text = stringResource(
+                        R.string.quick_calculated_on,
+                        DateFormatUtils.calculatedOn(quick.calculatedDate)
+                    ),
+                    color = ArkColor.TextTertiary,
+                    fontSize = 13.sp
```

Comment:
Is it fine if we do not set this fontSize? I think default value is good enough
```suggestion
```

Reply:
Then date will be the same size as calculations, it doesn’t look very good. I think we will discuss this with designer.

Reply:
Is there a generic way to solve such problem? Like not fixing exact font size, but something like `80%`. Or maybe something like title/subtitle/paragraph?..

Reply:
We can create our own text styles with color and font size, but I need a design system in Figma with these styles.